src/Docker.PowerShell/Docker.psm1

```diff
@@ -6,6 +6,7 @@
 # the module is in.
 #
 
+$paramBackup = $PSDefaultParameterValues.Clone()
```

Comment:
Are you trying to protect your code from a real customer reported issue or a theoretical issue?

Most modules don't bother doing anything with $PSDefaultParameterValues, and this module is very simple, very few conflicts to worry about.